kinto/authorization.py

```diff
@@ -139,10 +139,10 @@ def build_permissions_set(object_uri, unbound_permission,
 
 
 @implementer(IAuthorizationPolicy)
-class AuthorizationPolicy(cliquet_authorization.AuthorizationPolicy):
+class AuthorizationPolicy(core_authorization.AuthorizationPolicy):
```

Comment:
I like this idea, but in order to make it work, I have to change it to

```python
import kinto.core.authorization
from kinto import core
```

... because `core.authorization` is only defined as a side-effect of importing `kinto.core.authorization`. This is an unfortunate quirk of Python's import behavior. I don't think this is an improvement, and `flake8` doesn't like it either.